.circleci/config.yml

```diff
@@ -16,13 +21,13 @@ anchors:
         # Clone the submodules
         git submodule update --init
 
-        # Trust all of the .envrc files for direnv
-        find . -name '.envrc' -exec direnv allow {} \;
-
         # Update Yarn
         curl -o- -L https://yarnpkg.com/install.sh | bash
         echo 'export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"' >> "$BASH_ENV"
 
+        # Provide gulp
+        yarn global add gulp@4.0.0
```

Comment:
Generally `gulp-cli` is the package to install globally, and individual projects should locally depend on `gulp` -- this seems fishy.

Reply:
Right, very true.  Also I should have hesitated before adding anything globally; since the package.json already has gulp 4, I can just use `PATH=$(yarn bin):$PATH` instead. See changes.